guides/source/configuring.md

```diff
@@ -368,6 +368,9 @@ All these configuration options are delegated to the `I18n` library.
   has_many relationships to be displayed with an index as well as the error.
   Defaults to `false`.
 
+* `config.active_record.use_schema_cache_dump` enables `schema cache` in Rails, which allows you to get schema info from `db/schema_cache.yml` (generated by `bin/rails db:schema:cache:dump`) instead of sending query to database.
```

Comment:
enables users to save schema cache information in `db/schema_cache.yml` (generated by `bin/rails db:schema:cache:dump`), instead of having to send a query to the database to get this information.